### ISSUE TITLE
IPC Server Added

### ISSUE THREAD

**Issue by wh010ne** [ManlyMarco/RuntimeUnityEditor#7]
Basic IPC server added in fork https://github.com/wh010ne/RuntimeUnityEditor

Includes test client. For now just foundation functionality for IPC communication. Ready for tying to other components. Non-blocking, runs on background thread. Asynchronous. Use's native API. no dependencies.

**Comment by ManlyMarco** [comment]
Good work so far. Let me know if you add any more to this, I'll think about incorporating it then.

**Comment by wh010ne** [comment]
Good to know Marco. I've got some improvements for telnet, current IPC named pipes, and adding so far a working WCF foundation, I've also got Remoting covered covered, just have to add it into project still. But so far I've got pretty working foundations of them all, so know comes the fun part of implementing them. Also, working a good serialization method for some of Unity's complex objects. So far as a test, I've got an FPS counter going back and forth fine, but what if I try GameObject, PlayerSettings, etc... may have a trick to handle that though, still testing it.
